app/models/user.rb

```diff
@@ -67,6 +70,10 @@ def forget
     update(remember_digest: nil)
   end
 
+  def feed
+    Micropost.new_post.where("user_id = ?", id)
```

Comment:
Viết như này luôn được hôn?
```suggestion
   microposts.new_post
```

Reply:
- Không được vì khi viết microposts.new_post có nghĩa là đang gọi một instance method trên instance variable microposts

Reply:
Fen lại chén gió :v: 
Thử đi xem ra kết quả giống nhau không :)))